src/main/CriticalErrorHandler.ts

```diff
@@ -57,7 +57,7 @@ export default class CriticalErrorHandler {
             defaultId: 0,
         }).then(({response}) => {
             if (response === 0) {
-                throw new Error('BrowserWindow \'unresponsive\' event has been emitted');
+                app.relaunch();
```

Comment:
This didn't work, it would just throw an error and fail silently. So now we do what I would expect which is to relaunch the app.